### PR TITLE
add bio

### PR DESCRIPTION
**Pull request by kvenkman**

### CODE REVIEW

**Comment by scottyhq** [comment]
@jomey I'm not sure what the issue is here, but in the preview Karthik and Kehan's profiles seem to be getting merged ?!

File: team/karthik-venkataramani.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    - Machine Learning
+    - Geospatial Data analysis
+  social:
+    - icon: github
+      link: https://github.com/kvenkman
+    - icon: linkedin
+      link: https://www.linkedin.com/in/karthikvenkataramani/
+  user_groups:
+    # From https://escience.washington.edu/using-data-science/hackweeks/planning/hackweek-roles
+    - Helper
```
Comment (jomey) [inline team/karthik-venkataramani.yaml]: @scottyhq - Looks like we have an issue with the YAML here.
Should look like this:
```
  roles:
    - Helper
```

Comment (scottyhq) [reply team/karthik-venkataramani.yaml]: I don't think that's it

https://github.com/uwhackweek/event-page-2024/blob/099ce079e3ce59603b8e921567796408e3be561d/team/template.yaml#L20-L23

maybe some special character situation or subtle thing with the way individual files get merged...

Comment (jomey) [reply team/karthik-venkataramani.yaml]: > I don't think that's it
> 
> https://github.com/uwhackweek/event-page-2024/blob/099ce079e3ce59603b8e921567796408e3be561d/team/template.yaml#L20-L23
> 
> maybe some special character situation or subtle thing with the way individual files get merged...

Oh .. yeah. That can't be it.
Let me have another look

**Review by jomey — APPROVED** [review]

**Comment by scottyhq** [comment]
@kvenkman out of curiosity what code editor and OS were you using to create your file?

**Pull request merged**